source/api/commands/route2.md

```diff
@@ -0,0 +1,394 @@
+---
+title: route2
+containerClass: experimental
+---
+
+{% note warning %}
+{% fa fa-warning orange %} **This is an experimental feature. In order to use it, you must manually configure the {% url "`experimentalNetworkMocking`" experiments %} option to `true`.** See {% issue 687 %} for more details.
+{% endnote %}
+
+Use `cy.route2()` to manage the behavior of network requests at the network layer.
+
+# Comparison to `cy.route()`
+
+Unlike {% url `cy.route()` route %}, `cy.route2()`:
+
+- can mock all types of network requests including Fetch API, page loads, XMLHttpRequests, etc.
+- does not require calling {% url `cy.server()` server %} before use
+
+# Syntax
+
+```javascript
+// JavaScript
+cy.route2(url)
+cy.route2(url, response)
+cy.route2(method, url)
+cy.route2(method, url, response)
+```
+
+```typescript
+// TypeScript
+cy.route2(url: RouteMatcher, response?: RouteHandler): Chainable<null>
+cy.route2(method: string, url: RouteMatcher, response?: RouteHandler)): Chainable<null>
+```
+
+{% note info %}
+To learn more about types, check out [the repo](https://github.com/cypress-io/cypress/blob/0d60f7cd3ede4c5a79e151b646fa2377a7ddb16c/packages/net-stubbing/lib/external-types.ts#L209-L210) for more information.
+{% endnote %}
+
+## Usage
+
+**{% fa fa-check-circle green %} Correct Usage**
+
+```javascript
+cy.route2('/users/**')
+```
+
+## Arguments
+
+**{% fa fa-angle-right %} url**
```

Comment:
Going to leave it as `url` since this is what most users are familiar with, but there is an explicit definition of `Type` inside of it so the `routeMatcher` piece is not lost.